src/SFA.DAS.Employer.PR.Web/Program.cs

```diff
@@ -17,6 +17,7 @@
     .AddServiceRegistrations(configuration)
     .AddAuthenticationServices(configuration)
     .AddSession(configuration)
+    .AddMemoryCache(configuration)
```

Comment:
This is taken care of in AddSession Entension

Reply:
No I moved it out of session, as I felt it was better in its own extension and not tied to session, as we are introducing use of distributed cache to deal the the providers, so will be used by session and cache.

Reply:
Then need to move out distributed cache as well from there

Reply:
You could instead remove the session service registration from that extension and rename the extension to AddDistributedCache.